src/components/dialogs/contingency-list/creation/contingency-list-creation-form.tsx

```diff
@@ -25,6 +25,18 @@ import { useSelector } from 'react-redux';
 import { elementExists } from '../../../../utils/rest-api';
 import { ChangeEvent, FunctionComponent } from 'react';
 import { AppState } from 'redux/reducer';
+import Box from '@mui/material/Box';
+
+const styles = {
+    FillerContainer: {
```

Comment:
```suggestion
    fillerContainer: {
```
use camelCase, to refacto also in others styles